qiskit_sphinx_theme/pytorch_base/static/css/theme.css

```diff
@@ -9248,18 +9256,6 @@ button.bg-dark:focus {
   text-transform: capitalize !important;
 }
 
-.font-weight-light {
-  font-weight: 300 !important;
-}
-
-.font-weight-normal {
-  font-weight: 400 !important;
-}
-
-.font-weight-bold {
-  font-weight: 700 !important;
-}
-
```

Comment:
These are not used per `rg -g '**/*.html' -g '**/*.js' 'font-weight-italic'` in `docs/_build/html`. I can add them back if people prefer.